<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>568</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>998</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Controllers</string>
             </attribute>
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>190</y>
                <width>67</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>380</x>
                <y>10</y>
                <width>601</width>
                <height>451</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QTableWidget" name="tableWidgetListControllers"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="pushButtonModifyController">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Modify</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>408</width>
                     <height>18</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonEraseController">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Erase</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="layoutWidget_2">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>10</y>
                <width>309</width>
                <height>345</height>
               </rect>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="labelLastName">
                 <property name="text">
                  <string>Last name</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEditLastName"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="labelFirstName">
                 <property name="text">
                  <string>First name</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEditFirstName"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="labelStreet">
                 <property name="text">
                  <string>Street</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEditStreet"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="labelCity">
                 <property name="text">
                  <string>City</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="lineEditCity"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="labelZipCode">
                 <property name="text">
                  <string>ZIP code</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="lineEditZipCode"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="labelPhoneNumber">
                 <property name="text">
                  <string>NÂ° phone         </string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="lineEditPhoneNumber"/>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="labelMail">
                 <property name="text">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLineEdit" name="lineEditEmail"/>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="labelSSN">
                 <property name="text">
                  <string>Social Security Number</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QLineEdit" name="lineEditSSN"/>
               </item>
               <item row="8" column="1">
                <widget class="QPushButton" name="pushButtonAddController">
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Producers</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="labelProducers">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Producers&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>648</width>
                      <height>13</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTableWidget" name="tableWidgetListProducers"/>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_18">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_16">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_17">
                      <item>
                       <widget class="QLabel" name="label_3">
                        <property name="text">
                         <string>Meeting's propositions</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QComboBox" name="comboBoxListDate_2"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_5">
                      <property name="text">
                       <string>Accept</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_18">
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="font">
                       <font>
                        <family>STIXGeneral</family>
                        <italic>true</italic>
                       </font>
                      </property>
                      <property name="layoutDirection">
                       <enum>Qt::LeftToRight</enum>
                      </property>
                      <property name="autoFillBackground">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>Upon acceptance the producer will be notified of your visit  </string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_14">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>248</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_20">
                    <item>
                     <spacer name="horizontalSpacer_16">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>448</width>
                        <height>13</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Products</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_16">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_15">
                <item>
                 <spacer name="horizontalSpacer_12">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>928</width>
                    <height>38</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_14">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_9">
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_11">
                        <item>
                         <layout class="QVBoxLayout" name="verticalLayout_8">
                          <item>
                           <widget class="QLabel" name="label_9">
                            <property name="text">
                             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Shelves&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QTableWidget" name="tableWidgetRayonList">
                            <column>
                             <property name="text">
                              <string>Name</string>
                             </property>
                            </column>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_8">
                          <item>
                           <widget class="QLineEdit" name="lineEditCreateShelf"/>
                          </item>
                          <item>
                           <spacer name="horizontalSpacer_7">
                            <property name="orientation">
                             <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                             <size>
                              <width>78</width>
                              <height>13</height>
                             </size>
                            </property>
                           </spacer>
                          </item>
                          <item>
                           <widget class="QPushButton" name="pushButtonCreateRayon">
                            <property name="text">
                             <string>Create shelf</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <spacer name="verticalSpacer_3">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>38</width>
                          <height>188</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_12">
                        <item>
                         <layout class="QVBoxLayout" name="verticalLayout_9">
                          <item>
                           <widget class="QLabel" name="label_10">
                            <property name="text">
                             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Categories&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QTableWidget" name="tableWidgetCategoryList">
                            <column>
                             <property name="text">
                              <string>Name</string>
                             </property>
                            </column>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_7">
                          <item>
                           <widget class="QLineEdit" name="lineEditAddCategory"/>
                          </item>
                          <item>
                           <spacer name="horizontalSpacer_8">
                            <property name="orientation">
                             <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                             <size>
                              <width>78</width>
                              <height>13</height>
                             </size>
                            </property>
                           </spacer>
                          </item>
                          <item>
                           <widget class="QPushButton" name="pushButtonAddCategory">
                            <property name="text">
                             <string>Add category</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <spacer name="verticalSpacer_4">
                        <property name="orientation">
                         <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                         <size>
                          <width>38</width>
                          <height>188</height>
                         </size>
                        </property>
                       </spacer>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_13">
                        <item>
                         <layout class="QVBoxLayout" name="verticalLayout_10">
                          <item>
                           <widget class="QLabel" name="label_11">
                            <property name="text">
                             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Products&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QTableWidget" name="tableWidgetProductList"/>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_6">
                          <item>
                           <widget class="QLineEdit" name="lineEditAddProduct"/>
                          </item>
                          <item>
                           <spacer name="horizontalSpacer_9">
                            <property name="orientation">
                             <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                             <size>
                              <width>78</width>
                              <height>13</height>
                             </size>
                            </property>
                           </spacer>
                          </item>
                          <item>
                           <widget class="QPushButton" name="pushButtonAddProduct">
                            <property name="text">
                             <string>Add product</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>38</width>
                      <height>338</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="labelErrorMessage">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_11">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>918</width>
                    <height>98</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Propositions</string>
             </attribute>
             <widget class="QTableWidget" name="tableWidgetListOfProductPropositions">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>50</y>
                <width>601</width>
                <height>271</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>10</y>
                <width>191</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Products propositions</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_2">
              <property name="geometry">
               <rect>
                <x>680</x>
                <y>60</y>
                <width>95</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Accept</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_3">
              <property name="geometry">
               <rect>
                <x>680</x>
                <y>100</y>
                <width>95</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Refuse</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>988</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>838</width>
              <height>38</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
